paddle/fluid/operators/multinomial_op.cu

```diff
@@ -31,6 +32,14 @@ __global__ void NormalizeProbability(T* norm_probs, const T* in_data,
                                      T* sum_rows) {
   int id = threadIdx.x + blockIdx.x * blockDim.x +
            blockIdx.y * gridDim.x * blockDim.x;
+  PADDLE_ENFORCE(in_data[id] >= 0.0,
+                 "The input of multinomial distribution should be >= 0");
+  PADDLE_ENFORCE(
+      !std::isinf(static_cast<double>(in_data[id])) &&
```

Comment:
Please do not combine several logical expressions in one ENFORCE.

Reply:
done